### PR TITLE
YD-194 Removed the type from the ResponseMessageDTO

### PR DESCRIPTION
**Pull request by Bert-R**
This response is only used for errors, so the type is pointless.

### CODE REVIEW

File: core/src/main/java/nu/yona/server/rest/ResponseMessageDTO.java [hunk 1]
```diff
@@ -2,13 +2,9 @@
 
 /**
  * This DTO is used to communicate error messages or other type of messages to the client.
- * 
- * @author pgussow
  */
 public class ResponseMessageDTO
```
Comment (aristotelos) [inline core/src/main/java/nu/yona/server/rest/ResponseMessageDTO.java]: Consider to rename the class to `ErrorResponseDTO` or something like that.

**Comment by aristotelos** [comment]
Added one suggestion.

File: core/src/main/java/nu/yona/server/rest/ResponseMessageDTO.java (hunk 1 above)
Comment (Bert-R) [inline core/src/main/java/nu/yona/server/rest/ResponseMessageDTO.java]: Good one. Will do.

**Comment by Bert-R** [comment]
Done.

**Pull request merged**